es-vue-base/src/lib-components/EsPopover.vue

```diff
@@ -15,6 +15,7 @@
             -->
             <slot name="title" />
             <EsButton
+                :inline="true"
```

Comment:
Nit: for brevity, this could just be `inline` rather than `:inline="true"`. Vue generally matches the rules for HTML tag attributes (e.g. `<EsButton>` should really be `<es-button>` but we should really have a linting rule for that like we do on our other repos).